### DISCUSSION TITLE
Long time lapse for construction project

### DISCUSSION THREAD

**Discussion by jhalfter** [gopro/labs#238] [Q&A]
Hi! I want to set up a solar panel + powerbank + wifi modem with a GoPro 10 to capture images in an interval of 5min from 8.30am - 15pm and then have it upload automatically. 

I'm unsure how I can get a script to do this for me. 

This script seem to do what I need, but with video - how do I change it to photo? 
https://gopro.github.io/labs/control/dailytl/ script: cmd=!08:00NmTfLp.300!S!3600E!U!1R 

Or this
https://gopro.github.io/labs/control/set/?cmd=mPdP>08:29<15:30!285SQ~!08:30S!1R

And one more question 
Is it possible to combine two scripts? So I also can reset camera (not script) every day in case of failure
with this: 
!MNAKE=1!MWAKE=1!MBOOT="!Ldaily"!SAVEdaily=mTP>07:59<20:00!1SQ!15:30E!1U~!1U

Thank you - I hope you can help me  : )

**Comment by dnewman-gpsw** [comment]
Go with the first script, but change from timelapse (mT) to timelapse Photo (mTP), and make sure you have that preset in enabled in your camera (it is not there by default.)

So the script:  !08:30NmTPfLp.300!S!15:00EoW1!U!1R

If you combine with an external timer to power the camera off once each night.  This is the boot scripting needed

!MWAKE=2!MBOOT="!Ldaily"!SAVEdaily=!08:30NmTPfLp.300!S!15:00EoW1!U!1R
